package.json

```diff
@@ -25,12 +25,12 @@
   "homepage": "https://github.com/fastify/fastify-oauth2#readme",
   "devDependencies": {
     "fastify": "^2.0.1",
-    "nock": "^9.3.2",
+    "nock": "^12.0.3",
```

Comment:
Please restore this to v9, thanks.

Reply:
I reverted to v9. 

Fastify plan to drop node 6 and 8?  Currently, this plugin uses simple-oauth2 v1  that is no longer supported. Upgrading to v3 would be possible but require dropping v6.

Reply:
> Fastify plan to drop node 6 and 8?

Yes, we will release Fastify v3 soon.
Check out the main repo for more info